Shard management (#2322)
influxdata/docs-v2: content/influxdb/v2.0/reference/glossary.md

Related entries: [aggregator plugin](#aggregator-plugin), [input plugin](#input-plugin), [output plugin](#output-plugin), [processor plugin](#processor-plugin)

### shard

A shard contains encoded and compressed data for a specific set of [series](#series).

> A shard consists of multiple [TSM files](#tsm-time-structured-merge-tree) on disk.

Author: Updated

All points in a series in a given shard group are stored in the same shard (TSM file) on disk.
A shard belongs to a single [shard group](#shard-group).

For more information, see [Shards and shard groups (OSS)](/influxdb/%762.0/reference/internals/shards/).

Related entries: [series](#series), [shard duration](#shard-duration),
[shard group](#shard-group), [tsm](#tsm-time-structured-merge-tree)

### shard group

> Shard groups are logical containers for shards organized by [bucket](#bucket).

Author: Removing "organized by bucket." It's confusing. Shard groups belong to a bucket.

Every bucket that contains data has at least one shard group.
A shard group contains all shards with data for the time interval covered by the shard group.

> The interval spanned by each shard group is the [shard group duration](#shard-group-duration).

Reviewer: Does it make sense/streamline to combine lines 928-929 with line 926? Something like:

_Shard groups are organized by [bucket](#bucket) and contain all shards with data for a specified interval known as the [shard group duration](#shard-group-duration)._

For more information, see [Shards and shard groups (OSS)](/influxdb/%762.0/reference/internals/shards/).

Related entries: [bucket](#bucket), [retention period](#retention-period),
[series](#series), [shard](#shard), [shard duration](#shard-duration)

### shard group duration

> The duration of time each each [shard group](#shard-group) spans.
> The interval is determined by the `shard-group-duration` of the [bucket](#bucket).

Reviewer: ```suggestion
The duration of time or interval that each each [shard group](#shard-group) spans. Set the `shard-group-duration` for each [bucket](#bucket).
```

Reviewer: by spans, do we mean covers or exists?
